spec/conformance/mds_finder.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'fido_metadata'
+
+module WebAuthn
```

Comment:
Maybe better not to pollute the `WebAuthn` module with external stuff?

This may end up being a reference for users implementing their own finders, not sure we want to promote them using the gem namespace... :-P

Reply:
Agree!

Reply:
Cool, thanks!